internal/pkg/template/templates/workloads/partials/cf/secrets.yml

```
@@ -2,7 +2,12 @@
 Secrets:
 {{- range $name, $secret := .Secrets}}
 - Name: {{$name}}
-  ValueFrom: {{if not $secret.RequiresSub }}{{$secret.ValueFrom}}{{- else}} !Sub 'arn:${AWS::Partition}:{{$secret.Service}}:${AWS::Region}:${AWS::AccountId}:{{$secret.ValueFrom}}' {{- end }}
+{{- if $secret.RequiresImport}}
+  ValueFrom:
+    Fn::ImportValue: {{ quote $secret.ValueFrom -}}
```

Do we need this before delimiters for some reason here? As far as I know they are used to truncate the white spaces. We can remove them unless they are doing some operation here. Similarly for other templates as well 
```suggestion
    Fn::ImportValue: {{ quote $secret.ValueFrom }}
```